src/extension/interop/ExtensionInterop.cs

```diff
@@ -121,4 +121,7 @@ internal static partial class NativeMethods
 {
     [DllImport("thrive_extension")]
     internal static extern int ExtensionGetVersion(IntPtr thriveConfig);
+
+    [DllImport("thrive_extension")]
+    internal static extern int ArrayMeshUnwrap(IntPtr mesh, float texelSize);
```

Comment:
```suggestion
    internal static extern bool ArrayMeshUnwrap(IntPtr mesh, float texelSize);
```
Just noticed this return value mismatch with the C++ code.